impeller/scene/importer/scene.fbs

```diff
@@ -41,6 +43,17 @@ struct Vertex {
   color: Color;
 }
 
+enum IndicesType:byte {
+  k16Bit,
+  k32Bit,
+}
+
+table Indices {
+  data: [ubyte];
```

Comment:
I believe these arrays already encode their size. So the count would be redundant right?